### DISCUSSION TITLE
Dietpi-Software | Zabbix-agent(2)

### DISCUSSION THREAD

**Discussion by Ivo-tje** [MichaIng/DietPi#6069] [Software Request]
#### Formal software information
- Software title | Zabbix-agent(2)
- Short description | Used to monitor the device
- Official URL (if available) | https://www.zabbix.com/download
- GitHub URL (if available) | https://git.zabbix.com/projects/ZBX/repos/zabbix/browse
- (Official) install documentation | https://www.zabbix.com/documentation/6.2/en/manual/installation/install

#### Are there similar/alternative software titles available with DietPi-Software?
No, snmpd would be sort of alternative, but this is not available

#### What makes your requested software better than the above solutions, if available?
-

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. apt install zabbix-agent(2)
2. edit configuration file to set server variable
3. connect to zabbix-server

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
Can help with testing if needed, got a zabbix monitorin setup and a nano-pi to test on.

**Comment by MichaIng** [comment]
Many thanks for your suggestion.

I converted this into a discussion, so others can vote for it with the UP button on your post.
